src/components/Chip/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shape, string, node, element } from 'prop-types';
+import MuiChip from '@material-ui/core/Chip';
+
+function Chip({ classes, label, icon }) {
+  return (
+    <MuiChip
+      className={classes.chip}
+      label={label}
+      icon={icon}
+      size="small"
+      variant="outlined"
+    />
+  );
+}

```

Comment:
Presently, this component just imports the Chip component and then renders it. It doesn't add anything on top of what material-ui gives us but rather just forward the props. What do you think about the following (untested):

```jsx
import React from 'react';
import { shape, string, node, element } from 'prop-types';
import { withStyles } from '@material-ui/core/styles';
import MuiChip from '@material-ui/core/Chip';

const styles = theme => ({
  /**
   * Chips used for keyword notations.
   */
  chip: {
    color: theme.palette.text.primary,
    borderColor: theme.palette.text.secondary,
  },
});
function Chip({ classes, label, icon }) {
  return (
    <MuiChip
      className={classes.chip}
      label={label}
      icon={icon}
      size="small"
      variant="outlined"
    />
  );
}

Chip.propTypes = {
  label: node.isRequired,
  icon: element,
};

export default withStyles(styles)(React.memo(Chip));
```

Users will now be able to just do `<Chip ... />` rather than `<Chip classes={{ chip: {...}}} ... />`.

Reply:
Applied changes to move the chip classes to directly be defined and used within the `Chip` component. (used the [hook API](https://material-ui.com/styles/basics/#hook-api) instead of the Higher-order component API to define and apply the style since we use functional components and the hook API for other components as well) Now, the users do not need to pass the chip class every time :)